packages/hoc-disabled-tooltip/src/disabledTooltip.js

```diff
@@ -5,6 +5,15 @@ import Tooltip from "@crave/farmblocks-tooltip";
 
 import Wrapper from "./styledComponents/Wrapper";
 
+export const newPropTypes = {
```

Comment:
hmmmm.... why `newPropTypes`? 
Can we change to something like `disabledTooltipProps`?

Reply:
yes, your suggestion is better